tests/foreman/cli/test_subscription.py

```diff
@@ -284,3 +284,29 @@ def test_positive_auto_attach_disabled_golden_ticket(
     with pytest.raises(CLIReturnCodeError) as context:
         target_sat.cli.Host.subscription_auto_attach({'host-id': host_id})
     assert "This host's organization is in Simple Content Access mode" in str(context.value)
+
+
+@pytest.mark.tier2
+def test_positive_prepare_for_sca_only_hammer(function_org, target_sat):
+    """Verify that upon certain actions, Hammer notifies users that Simple Content Access
+        will be required for all organizations in the next release
+
+    :id: d9985a84-883f-46e7-8352-dbb849dbfa34
+
+    :expectedresults: Hammer returns a message notifying users that Simple Content Access
+        will be required for all organizations in the next release
+    """
+    org_results = target_sat.execute(
+        f'hammer organization update --id {function_org.id} --simple-content-access false'
+    )
+    assert (
+        'Simple Content Access will be required for all organizations in the next release'
+        in str(org_results.stderr)
+    )
+    sca_results = target_sat.execute(
+        f'hammer simple-content-access disable --organization-id {function_org.id}'
```

Comment:
Can we update it to use robottelo cli method SimpleContentAccess.disable()?

Reply:
@Gauravtalreja1 simplecontentaccess.disable() just returns an empty string. I cannot get the required output message unless I pull from robottelo.log

Reply:
@ColeHiggins2 as you need to verify output from command's stderr, and you're not able to use cli.SimpleContentAccess for this because of `ignore_stderr=True` argument, you could remove it from here and update the PR for the same https://github.com/SatelliteQE/robottelo/blob/master/robottelo/cli/simple_content_access.py#L33

Reply:
fixed